src/cavendish_particle_tracks/_widget.py

```diff
@@ -404,13 +414,36 @@ def _on_click_load_data(self) -> None:
             self.msg.show()
             return
 
-        for subdir, stack_name in zip(
-            folder_subdirs, ["stack1", "stack2", "stack3"]
-        ):
-            stack = imread(subdir + "/*")
-            self.viewer.add_image(stack, name=stack_name)
-            # TODO: investigate the multiscale otption.
-        self.viewer.dims.axis_labels = ("Event", "Y", "X")
+        def crop(array):
+            # Crops view 1 and 2 to same size as view 3 by removing whitespace
+            # on left, as images align on the right.
+            return array[:, :, -8377:, :]
```

Comment:
-8377 is a magic number... ?

Reply:
Hmm. No way to grabe the dimension of the smallest or something?
At the worst, you probably want say what it is by giving it an explicit variable name:
```py
magic_number_smallest_view_pixels = -8377
return array[:, :, magic_number_smallest_view_pixels:, :]
```

... just because in 2 weeks time, no one will remember what 8377 is.